### PR TITLE
Create Reckitt Benckiser

### PR DESCRIPTION
**Pull request by EpLiar**

### CODE REVIEW

**Comment by Loyalsoldier** [comment]
Please finish a PR first at your fork repo and then file a pull request. Every time you push a commit to a draft PR, repo watchers will receive a notification and email, which may disturbs sometimes.

**Comment by EpLiar** [comment]
@Loyalsoldier I'll pay attention next time.

The reason I file pull requests and mark them as draft is that it takes a long time to complete.

**Comment by Loyalsoldier** [comment]
> @Loyalsoldier I'll pay attention next time.
> 
> The reason I file pull requests and mark them as draft is that it takes a long time to complete.

Actually, emails and notifications will be sent to repo watchers every time a new commit is pushed to an open PR including draft PR. So I suggest that we'd better finish it first, and then file a PR.

**Comment by Loyalsoldier** [comment]
And please squash commits in a PR. For example, you `create finish` in commit 04d0f61 then you `update finish` in commit e69a09e

IMO, we should squash these two commit into one as long as it does the same thing(add `finish` domains into this PR in this example) for cleaner and better commit log.

**Comment by EpLiar** [comment]
> we should squash these two commit into one as long as it does the same thing(add finish domains into this PR in this example) for cleaner and better commit log.

I should be more careful next time. But it's hard to ensure that there are no mistakes in the commit.

**Comment by Loyalsoldier** [comment]
可能用英文讲得不是特别清楚，我用中文完整描述一遍。

1. 在工作完成后，再提交 pull request 以避免期间不断修改和添加 commit，过多通知和邮件可能会对 repo watchers 造成不必要的打扰
2. 对完成相同事情的多个 commits 进行 squash，整合成一个 commit，使得项目的 commit log 更加清晰。以这个 PR 里的 04d0f61 和 e69a09e 两个 commits 举例：都是为 `finish` 这个列表添加域名，应该通过 rebase 和 squash 整合成一个 commit
3. 提交 PR 后，发现有错误，或者 review 过程中，reviewer 提出了修改意见和建议，可以通过 force push 的方式依然保持 commit 的清晰和整洁。

2 和 3 分别是 git 的“进阶”和“危险”操作，应该学习一下。最后再次感谢各位 contributors 🎉

**Comment by Loyalsoldier** [comment]
@EpLiar 

I suggest we leave this PR open util you have learned git interactive rebase, squash, and merge, and apply these git operation to this PR to make a cleaner commit logs.

**Comment by EpLiar** [comment]
@

> @EpLiar
> 
> I suggest we leave this PR open util you have learned git interactive rebase, squash, and merge, and apply these git operations to this PR to make a cleaner commit logs.

Agree.

File: data/calgoncarbon [hunk 1]
```diff
@@ -0,0 +1,6 @@
+calgon.cn
+calgon.com
```
Comment (Robot-DaneelOlivaw) [inline data/calgoncarbon:2]: There are multiple countries domains listed in [calgon.com](https://www.calgon.com/). Please take some time to add them.

Comment (EpLiar) [reply data/calgoncarbon:6]: > There are multiple countries domains listed in [calgon.com](https://www.calgon.com/). Please take some time to add them.

Done. Thanks to remind me.

File: data/movefree [hunk 2]
```diff
@@ -0,0 +1,6 @@
+keepmovingwithmovefree.com
+move-free.net
+movefree.cn
+movefrees.com
```
Comment (Robot-DaneelOlivaw) [inline data/movefree:4]: [movefree.com](movefree.com) redirects to [Schiff Vitamins](https://www.schiffvitamins.com/pages/move-free). Would be nice to include that.

Comment (EpLiar) [reply data/movefree:4]: Fixed.

**Pull request merged**